subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -616,6 +616,7 @@ static void isr_aux_setup(void *param)
 static int isr_rx(struct lll_sync *lll, uint8_t node_type, uint8_t crc_ok, uint8_t rssi_ready,
 		  enum sync_status status)
 {
+	uint8_t sched = 0U;
```

Comment:
```suggestion
	bool sched = false;
```

And update the following uses of this to true/false as well.

Reply:
Fixed.